### ISSUE TITLE
"unsupported geography heirarchy" error for geography = 'state'

### ISSUE THREAD

**Issue by katrinleinweber** [christopherkenny/censable#6]
According to the function documentation in [build_dec.R](https://github.com/christopherkenny/censable/blob/612668a7559bf27960e5d3637878524c5d348af1/R/build_dec.R#L36-L37) & [build_acs.R](https://github.com/christopherkenny/censable/blob/612668a7559bf27960e5d3637878524c5d348af1/R/build_acs.R#L35-L36), `state` is a valid value, but it can't actually be used in any such call:

```
build_dec(geography = 'state', state = 'DE')
build_acs(geography = 'state', state = 'DE')
```

@christopherkenny Is that expected?

To get the total population for a states, is therefore necessary to first download it with `geography = 'county'` and then summarize? I essentially only need this kind of output :

state | pop
-|-
AL | …
… | …

**Comment by christopherkenny** [comment]
Hi @katrinleinweber. Thanks for finding that issue! It was not expected. It's now fixed in version 0.0.6, which can be installed with `remotes::install_github('christopherkenny/censable')` or `pak::pak('christopherkenny/censable')` depending on your preferred installed.

**Comment by christopherkenny** [comment]
For additional clarity, if you need all states and `geography = 'state'`, you can use:

```
build_dec(geography = 'state')
```
The same will work for `build_acs()`.
